docs/self-managed/concepts/access-control/user-restrictions.md

```diff
@@ -0,0 +1,19 @@
+---
+id: user-restrictions
+title: "User restrictions"
+sidebar_label: "User restrictions"
+description: "Control the level of access a user or group has to perform tasks in the system via user restrictions."
+---
+
+:::caution
+User restrictions are enabled by default and can be disabled using environment variables. This feature should be enabled in the required component, see [Identity feature flags](../../../../self-managed/identity/deployment/configuration-variables/#feature-flags).
+:::
+
+User restrictions allow you to control the level of access a [user](/self-managed/identity/user-guide/roles/add-assign-role.md) or
```

Comment:
The only thing I am missing here is an example, so, what _kind_ of restriction can I place on a user or group? Even just a screenshot could be sufficient here, but if it is simply they can or cannot access certain tasks, clarifying that could be good 👍 We just say the "level" of access, so I'm largely curious what kinds of levels we are talking about.